resources/views/forum/topics/show.blade.php

```diff
@@ -328,22 +328,41 @@ class="btn-circle btn-circle--topic-nav"
                     </span>
                 </a>
 
-                <button
-                    type="button"
-                    class="btn-osu-big btn-osu-big--forum-reply js-forum-topic-reply--stick"
-                >
-                    <span class="btn-osu-big__content">
-                        <span class="btn-osu-big__icon">
-                            <i class="fas fa-comment"></i>
+                @if (Auth::check())
+                    <button
+                        type="button"
+                        class="btn-osu-big btn-osu-big--forum-reply js-forum-topic-reply--stick"
+                    >
+                        <span class="btn-osu-big__content">
+                            <span class="btn-osu-big__icon">
+                                <i class="fas fa-comment"></i>
+                            </span>
+
+                            <span class="btn-osu-big__left">
+                                <span class="btn-osu-big__text-top">
+                                    {{ trans('forum.topics.actions.reply') }}
+                                </span>
+                            </span>
                         </span>
+                    </button>
+                @else
+                    <a
+                        href="{{ route('forum.topics.show', $topic) }}"
+                        class="btn-osu-big btn-osu-big--forum-reply js-forum-topic-reply--stick js-login-required--click"
```

Comment:
probably should just use `js-user-link` (and type button) without the topic reply js at all (it'll still reload once logged in). Also use `fa-sign-in-alt` icon like everywhere else.